src/schema/rules/tabular_data/modality_agnostic.yaml

```diff
@@ -18,6 +18,42 @@ Participants:
   index_columns: [participant_id]
   additional_columns: allowed
 
+Phenotypes:
+  selectors:
+    - extension == ".tsv"
+  initial_columns:
+    - participant_id
+  columns:
+    participant_id:
+      level: required
+      description_addendum: |
+        MUST be the first column in the file.
+        Note that data for one participant MAY be represented across multiple rows
+        in case of multiple sessions or runs, and
+        therefore the entry in the `participant_id` column will be repeated.
+    session_id:
+      level: optional
+      description_addendum: |
+        REQUIRED if sessions are defined in the dataset.
+        A `session_id` column MUST be added to all tabular files in the phenotype directory
+        as soon as multiple sessions are present in the data set
+        regardless of whether those sessions are in the
+        `phenotype/` data, `sub-<label>/` data, or a combination of the two.
+    run:
+      level: optional
+      description_addendum: |
+        REQUIRED if there are multiple runs within any session.
+        A chronological `run` number is used when
+        a measurement tool or assessment described by a tabular file
+        was repeated within a session.
+    acq_time:
+      level: optional
+      description_addendum: |
+        If acquisition time is available, the `acq_time` column CAN be used
+        to record the time of acquisition of each row in the tabular file.
+  index_columns: [participant_id, session_id, run, acq_time]
+  additional_columns: allowed
+
```

Comment:
@effigies or @rwblair: What's wrong with this picture? Please help oh lads of the schema.